hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
could you add a comment about why is this file needed?

Reply:
Content in this file is added as header in each generated file. I will add the details on how to use the script for code generation.